l1-contracts/package.json

```diff
@@ -76,7 +76,7 @@
     "token-info": "ts-node scripts/token-info.ts",
     "deploy-testkit": "ts-node scripts/deploy-testkit.ts",
     "verify": "hardhat run --network env scripts/verify.ts",
-    "verify-l2-contracts": "ts-node scripts/verify-l2-contracts.ts",
+    "verify-on-explorer": "yarn ts-node scripts/verify-on-explorer.ts",
```

Comment:
but you only verify l2 contracts, maybe say "verify-on-l2-explorer"? Otherwise it is not clear which chain do we verify on